uds/message/service_identifiers.py

```
@@ -187,6 +187,10 @@ def is_response_sid(cls, value: int) -> bool:
         return False
 
 
+SidAlias = Union[RequestSID, ResponseSID, int]
```

Ruff (0.15.20)

[warning] Use `X | Y` for type annotations

(UP007)